src/domains/dashboard/components/head-main.tsx

```diff
@@ -25,6 +27,7 @@ export const HeadMain = ({
         <ChartWrapper
           attributes={{
             ...commonAttributes,
+            host,
```

Comment:
can we please add `host` to commonAttributes?

Reply:
Done